mypy.ini

```diff
@@ -399,9 +398,6 @@ disallow_untyped_calls = False
 [mypy-arrow.*]
 ignore_missing_imports = True
 
-[mypy-cached_property.*]
-ignore_missing_imports = True
-
```

Comment:
Thanks to `typing-copilot` here. I remembered to remove the `untyped_decorators` check above, but I completely forgot this one.